mlflow/utils/rest_utils.py

```diff
@@ -77,6 +95,37 @@ def http_request(
     Returns:
         requests.Response object.
     """
+    if host_creds.databricks_workspace_client:
+        try:
+            if method == "GET":
+                query = kwargs.get("params") or kwargs.get("json")
+                extra_kwargs = {"query": query}
+            else:
+                extra_kwargs = {"body": kwargs.get("json")}
+
+            raw_response = host_creds.databricks_workspace_client.api_client.do(
```

Comment:
Is the SDK team aware of this?

Reply:
Yes `api_client.do`  is what they recommend in MLFlow usecase